dashboard-for-desktop/articles/dashboard-designer/appearance-customization/conditional-formatting.md

```diff
@@ -32,63 +21,80 @@ Comparison rules used in conditional formatting can be divided into the followin
 * [Bar Color Ranges](conditional-formatting/bar-color-ranges.md) - Allows you to visualize numeric values using bars whose colors are contained in the specified color set.
 * [Bar Gradient Ranges](conditional-formatting/bar-gradient-ranges.md) - Allows you to visualize numeric values using bars whose colors are contained in the specified color gradient.
```

Comment:
Please see my similar comment above. If my understanding is correct, would it be technically correct to say:

[Bar Gradient Ranges](conditional-formatting/bar-gradient-ranges.md) - Displays numeric values as bars. Values within a range display a specific bar gradient.

Reply:
Please change to:

* [Bar Gradient Ranges](conditional-formatting/bar-gradient-ranges.md) - Visualize numeric values as bars. Values within a range display a specific bar gradient.